src/types/slash.ts

```diff
@@ -50,7 +50,7 @@ export interface SlashCommandChoice {
   /** (Display) name of the Choice */
   name: string
   /** Actual value to be sent in Interaction */
-  value: string
+  value: any

```

Comment:
any?

Reply:
Value depends on the Command Option type.

Reply:
Okay